### PR TITLE
add support for Namron 4512758 thermostat

### PR DESCRIPTION
**Pull request by ttyridal**

### CODE REVIEW

File: src/devices/namron.ts [hunk 1]
```diff
@@ -1097,6 +1111,128 @@ const definitions: DefinitionWithExtend[] = [
             await reporting.currentPositionLiftPercentage(endpoint);
         },
     },
+    {
+        zigbeeModel: ['4512758'],
+        model: '4512758',
+        vendor: 'Namron AS',
```
Comment (Koenkk) [inline src/devices/namron.ts:1117]:
```suggestion
        vendor: 'Namron',
```

File: src/devices/namron.ts [hunk 2]
```diff
@@ -1097,6 +1111,128 @@ const definitions: DefinitionWithExtend[] = [
...
+        description: 'Namron Zigbee Termostat 16A',
```
Comment (Koenkk) [inline src/devices/namron.ts:1118]:
```suggestion
        description: 'Zigbee thermostat 16A',
```

**Comment by Koenkk** [comment]
Thanks!

**Comment by ttyridal** [comment]
Thanks, very efficient review !

 did the picture thing earlier in the week. It's already live 😁

**Pull request merged**